src/Infrastructure.VS/SpanTranslator.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.Text;
+
+namespace SonarLint.VisualStudio.Infrastructure.VS
+{
+    public interface ISpanTranslator
+    {
+        /// <summary>
+        /// Testable wrapper for <see cref="SnapshotSpan.TranslateTo"/>
+        /// </summary>
+        SnapshotSpan TranslateTo(SnapshotSpan original, ITextSnapshot targetSnapshot, SpanTrackingMode spanTrackingMode);
+    }
+
+    public class SpanTranslator : ISpanTranslator
+    {
+        public SnapshotSpan TranslateTo(SnapshotSpan original,
+            ITextSnapshot targetSnapshot,
+            SpanTrackingMode spanTrackingMode) =>
+            original.TranslateTo(targetSnapshot, SpanTrackingMode.EdgeExclusive);
```

Comment:
Since this is a one-liner wrapper, I'm not sure it's worth writing tests for it, it would be testing VS' internal framework.